### PR TITLE
Tb stats

### LINKED ISSUE #6: User can view Stats
As a user, when I visit the dashboard, I want to click the "Stats" link, so I can view my historical mood data 

- [ ] Create stats view
- [ ] Create stats w/ option to render as line graph, pie chart, and filter by timeframe (chart.js ?)

### PR DESCRIPTION
**Pull request by TravisBorgsmiller**
## What functionality does this accomplish?
- Builds out stats view
closes #6 
**Description:**
Adds a pie chart with chart.js that uses an activerecord query. 
### Test Coverage Percentage: x%
- [x] No tests have been changed
- [ ] Some tests have been changed
- [ ] All of the tests have been changed(Please describe what in the world happened):
### Rubocop:
- [x] No offenses
- [ ] Offenses (number of offenses: _ )
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [x] I have partially tested my code (please explain why): 
No sad path testing.
## Helpful Resources:
* Resource link AND small description of info gathered/learned

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -38,6 +38,8 @@ gem 'omniauth-google-oauth2'
 gem 'bootstrap', '~> 4.5.0'
 gem 'jquery-rails'
 gem 'figaro'
+gem "chartkick"
```
Comment (danielbldr) [inline Gemfile:41]: Good choice of gem for the chart

File: app/models/user.rb [hunk 2]
```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
   validates :name, presence: true
   validates :email, uniqueness: true
   validates :uid, uniqueness: true
+  has_many :user_moods
```
Comment (javier-aguilar) [inline app/models/user.rb:5]: this duplicates line 6

File: db/seeds.rb [hunk 3]
```diff
@@ -5,6 +5,15 @@
...
-Mood.create(name: "worried")
-Mood.create(name: "neutral_face")
-Mood.create(name: "smile")
+Mood.destroy_all
+User.destroy_all
+UserMood.destroy_all
+user = User.create!(name: 'Tj', email: 'tj@example.com', uid: '12345')
+mood1 = Mood.create!(name: 'worried')
+mood2 = Mood.create!(name: 'neutral_face')
+mood3 = Mood.create!(name: 'smile')
```
Comment (javier-aguilar) [inline db/seeds.rb:14]: The seed won't work since it doesn't have the rating

File: spec/features/user/view_stats_spec.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    mood5 = Mood.create(name: 'smile')
+    mood_entry1 = UserMood.create(user_id: user.id, mood_id: mood1.id, created_at: 'Tue 26 May 2020 21:21:12 UTC +00:00', updated_at: 'Tue 26 May 2020 21:21:12 UTC +00:00')
+    mood_entry2 = UserMood.create(user_id: user.id, mood_id: mood2.id, created_at: 'Wed 27 May 2020 21:21:12 UTC +00:00', updated_at: 'Wed 27 May 2020 21:21:12 UTC +00:00')
+    mood_entry3 = UserMood.create(user_id: user.id, mood_id: mood3.id, created_at: 'Thu 28 May 2020 21:21:12 UTC +00:00', updated_at: 'Thu 28 May 2020 21:21:12 UTC +00:00')
+    mood_entry4 = UserMood.create(user_id: user.id, mood_id: mood4.id, created_at: 'Fri 29 May 2020 21:21:12 UTC +00:00', updated_at: 'Fri 29 May 2020 21:21:12 UTC +00:00')
+    mood_entry5 = UserMood.create(user_id: user.id, mood_id: mood5.id, created_at: 'Sat 30 May 2020 21:21:12 UTC +00:00', updated_at: 'Sat 30 May 2020 21:21:12 UTC +00:00')
+
+    expect(current_path).to eq('/dashboard')
+    find(:xpath, "//a[@href='/stats']").click
+    assert_css('.chartjs-render-monitor') if has_css?('.chartjs-render-monitor', visible: :all)
```
Comment (SMJ289) [inline spec/features/user/view_stats_spec.rb:24]: That is an interesting way to click on a link that I have never seen!

File: app/views/stats/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<h1>Stats</h1>
+
+<%= pie_chart current_user.mood_query %>
```
Comment (SMJ289) [inline app/views/stats/index.html.erb:3]: Is this all we need in this view? It seems so simple!

**Pull request merged**